run-tests.sh

```diff
@@ -24,11 +24,7 @@ function cleanup() {
 trap cleanup EXIT
 
 
-python -m check_manifest --ignore ".*-requirements.txt"
+python -m check_manifest
 python -m sphinx.cmd.build -qnNW docs docs/_build/html
-eval "$(docker-services-cli up --db ${DB:-postgresql} --search ${SEARCH:-elasticsearch} --cache ${CACHE:-redis} --env)"
-invenio index init
+eval "$(docker-services-cli up --db ${DB:-postgresql} --search ${SEARCH:-opensearch} --env)"
```

Comment:
because the tests are also working without that. i prefer only to have what is necessary!

Reply:
Do the tests execute the exact same way without that flag though?
Because, while I agree with `only what is necessary`, another consideration is keeping tests as close as possible to production environment, right?

Reply:
I did a quick `grep` to see where the `redis`-cache is used and afaict it's used only for celery and caching sessions/REST-calls. So removing this is the right call.
The codebase is getting cleaner by the day, nice!